packages/ui/webauthn-pcd-ui/src/CardBody.tsx

```diff
@@ -6,7 +6,7 @@ import {
   styled
 } from "@pcd/passport-ui";
 import { PCDUI } from "@pcd/pcd-types";
-import { WebAuthnPCD } from "@pcd/webauthn-pcd/src/WebAuthnPCD";
+import { WebAuthnPCD } from "@pcd/webauthn-pcd";
```

Comment:
👍 

is there a way to prevent this type of mistake via a linter?

Reply:
I think so. Created an issue here https://github.com/proofcarryingdata/zupass/issues/1401